_chapters/js/style.md

```diff
@@ -1,6 +1,45 @@
 ---
-title: Style
+title: Style / Linting
 slug: js-style
 ---
-We currently recommend adhering to the [AirBnB](https://github.com/airbnb/javascript) JavaScript style guide. 
-Your team may, of course, choose a different style for your project if you all agree that you have reason to do so.
+We recommend adhering to the [Airbnb JavaScript style guide](https://github.com/airbnb/javascript).
```

Comment:
Can we go back to `We currently recommend`? As much as we may agree to this style guide for now, we know with most frontend conventions nowadays  it may change in the near or far future.